Application/EdFi.Ods.Standard/Artifacts/MsSql/Structure/Ods/1305-AuthViewsStaffUSIToEducationOrganizationId.sql

```diff
@@ -0,0 +1,25 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+IF OBJECT_ID('auth.StaffUSIToEducationOrganizationId', 'V') IS NOT NULL
+    DROP VIEW auth.StaffUSIToEducationOrganizationId
+GO
+
+
+CREATE VIEW auth.StaffUSIToEducationOrganizationId AS
```

Comment:
We have minimum sql 2016 support now, so we can use CREATE OR ALTER VIEW here instead of doing two statements, one dropping it and one creating it, more like how PostgreSQL is written.